### PR TITLE
fix: Minor changes to finish multitenancy TODO items

### PR DESCRIPTION
**Pull request by KShivendu**
## Summary of change

Minor changes to finish multitenancy TODO items

## Checklist for important updates

-   [ ] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `supertokens_python/constants.py`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `setup.py`
    -   In `supertokens_python/constants.py`
-   [ ] Had installed and ran the pre-commit hook
-   [ ] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If have added a new web framework, update the `supertokens_python/utils.py` file to include that in the `FRAMEWORKS` variable
-   [ ] If added a new recipe that has a User type with extra info, then be sure to change the User type in supertokens_python/types.py
-   [ ] Make sure that `syncio` / `asyncio` functions are consistent.

### CODE REVIEW

File: supertokens_python/recipe/dashboard/utils.py [hunk 1]
```diff
@@ -101,22 +101,24 @@ def from_user(
...
         self.first_name = first_name
         self.last_name = last_name
 
-        self.user_id = user_dict["user_id"]
-        self.recipe_id = user_dict.get("recipe_id")
-        self.time_joined = user_dict["time_joined"]
-        self.email = user_dict.get("email")
-        self.phone_number = user_dict.get("phone_number")
+        self.user_id = user_obj_dict["user_id"]
+        # self.recipe_id = user_obj_dict.get("recipe_id")
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/dashboard/utils.py:112]: why this comment?

File: supertokens_python/recipe/session/framework/fastapi/__init__.py [hunk 2]
```diff
@@ -22,11 +22,12 @@
 from supertokens_python.exceptions import SuperTokensError
 from supertokens_python.types import MaybeAwaitable
 from fastapi.responses import JSONResponse
-from fastapi import Request
 
 from ...interfaces import SessionContainer, SessionClaimValidator
 from supertokens_python.utils import set_request_in_user_context_if_not_defined
 
+from fastapi import Request
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/session/framework/fastapi/__init__.py:29]: why is this imported twice in this file?

File: supertokens_python/recipe/session/framework/fastapi/__init__.py [hunk 3]
```diff
@@ -41,7 +42,8 @@ def verify_session(
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[..., Coroutine[Any, Any, Union[SessionContainer, None]]]:
     _ = user_context
-    from fastapi import Request
+
+    from fastapi import Request  # pylint: disable=reimported
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/session/framework/fastapi/__init__.py:46]: why this `# pylint: disable=reimported`? Just remove the import from top?

File: supertokens_python/recipe/thirdparty/providers/facebook.py [hunk 4]
```diff
@@ -73,6 +73,11 @@ def Facebook(input: ProviderInput) -> Provider:  # pylint: disable=redefined-bui
     if input.config.user_info_map is None:
         input.config.user_info_map = UserInfoMap(UserFields(), UserFields())
 
+    if input.config.user_info_map.from_user_info_api is None:
+        input.config.user_info_map.from_user_info_api = UserFields()
+    if input.config.user_info_map.from_id_token_payload is None:
+        input.config.user_info_map.from_id_token_payload = UserFields()
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/thirdparty/providers/facebook.py:79]: why is this required here? given that we aren't adding anything to this in this provider?

Comment (rishabhpoddar) [reply supertokens_python/recipe/thirdparty/providers/facebook.py:79]: This comment applies to other providers as well.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: supertokens_python/recipe/session/framework/fastapi/__init__.py (hunk 3 above)
Comment (KShivendu) [reply supertokens_python/recipe/session/framework/fastapi/__init__.py:46]: Fixed.

File: supertokens_python/recipe/session/framework/fastapi/__init__.py (hunk 2 above)
Comment (KShivendu) [reply supertokens_python/recipe/session/framework/fastapi/__init__.py:29]: Fixed.

File: supertokens_python/recipe/thirdparty/providers/facebook.py (hunk 4 above)
Comment (KShivendu) [reply supertokens_python/recipe/thirdparty/providers/facebook.py:79]: Fixed.

File: supertokens_python/recipe/dashboard/utils.py (hunk 1 above)
Comment (KShivendu) [reply supertokens_python/recipe/dashboard/utils.py]: Fixed
